test/Common/DurableClientBaseTests.cs

```diff
@@ -57,7 +57,6 @@ public async Task StartNewAsync_is_calling_overload_method_with_specific_instanc
 
         [Theory]
         [Trait("Category", PlatformSpecificHelpers.TestCategory)]
-        [InlineData("")]
```

Comment:
Is this no longer an invalid value?

Reply:
It is not, we assign a guid to instanceId when the string is null or empty. (DurableClient line 111)